sgf-viewer/src/css/app.css

```diff
@@ -1,27 +1,26 @@
+:root {
+  --accent-color-1: #0086da;
```

Comment:
May want to update these & other colors to match [brand guidelines](https://docs.google.com/document/d/1RYEhcJwhDXe4nu0bhqukuh_LPbHWbmYcVK38mC4BnjY/edit) -- though these will change soon with the rebrand so not worth spending more than ~15 minutes on